roles/base/tasks/redhat_tasks.yml

```diff
@@ -3,6 +3,7 @@
     update_cache: true
     name: '*'
     state: latest
+  ignore_errors: yes
```

Comment:
errors here shouldn't be ignored; if you were having trouble let's discuss why that's not working.